build-system/tasks/e2e/describes-e2e.js

```diff
@@ -335,38 +339,43 @@ class ItConfig {
 }
 
 /**
- * Reports code coverage data to an aggregating endpoint.
- * @param {!Object} env e2e driver environment
+ * Extracts code coverage data from the page and aggregates it with other tests.
+ * @param {!Object} env e2e driver environment.
  * @return {Promise<void>}
  */
-async function reportCoverage(env) {
-  // Calling code only needs to wait until we've extracted the coverage data
-  // from the window, then it can begin cleaning up/resetting the state. POSTing
-  // the coverage data can happen asynchronously and doesn't need to block
-  // other testing.
-  const cov = await env.controller.evaluate(() => window.__coverage__);
-  if (cov) {
-    return new Promise((resolve, reject) => {
-      const req = http.request(
-        {
-          host: HOST,
-          port: PORT,
-          path: COV_REPORT_PATH,
-          method: 'POST',
-          headers: {'Content-type': 'application/json'},
-        },
-        (res) => {
-          res.on('data', () => {});
-          res.on('end', resolve);
-          res.on('error', reject);
-        }
-      );
-      req.write(JSON.stringify(cov));
-      req.end();
-    });
+async function collectCoverage(env) {
```

Comment:
Naming nit: Would it make sense to call this function something like `updateCoverage` or `appendCoverage` to clearly indicate that it's going to incrementally add to existing data and not re-collect everything?

Reply:
Done